### PR TITLE
Breaking: drop support for Node < 10

### PR DESCRIPTION
**Pull request by kaicataldo**
Small PR to update dependencies and drop support for Node < 10.

Does this seem okay? We could keep support for Node 8, but since it will be out of maintenance mode soon I don't think it's worth it.

### CODE REVIEW

**Review by mysticatea — DISMISSED** [review]

**Review by platinumazure — DISMISSED** [review]

**Review by nzakas — DISMISSED** [review]
LGTM

File: projects.yml [hunk 1]
```diff
@@ -25,60 +24,42 @@
...
     - tools
-    - --rule=indent:off
-    - --rule=no-multi-spaces:[error, {ignoreEOLComments:true}]
-    - --rule=timer-arguments:off
-    - --rule=no-self-assign:[error, {props:false}]
 
 - name: redux
   repo: https://github.com/reactjs/redux
-  commit: ac7cd32a5270eb374611bf73d072e84a1e7567c2
+  commit: 9819b5ef2909cacda0f09c8d29fde229bd7da5fa
```
Comment (kaicataldo) [inline projects.yml:44]: Updated to ESLint v6 here: https://github.com/reduxjs/redux/pull/3698

File: projects.yml [hunk 2]
```diff
@@ -25,60 +24,42 @@
 
 - name: mocha
   repo: https://github.com/mochajs/mocha
-  commit: 09ce746aa925d35317f2624fd36c77a31bb68e24
+  commit: 883ae4b5e1aacd8cf30694da33b391ce58f4cca8
```
Comment (kaicataldo) [inline projects.yml:27]: Updated to ESLint v6 here: https://github.com/mochajs/mocha/pull/4138

File: projects.yml [hunk 3]
```diff
@@ -25,60 +24,42 @@
...
-  repo: https://github.com/vuejs/vue
-  commit: d982e344b39391fe91b6dd91d51b2f0310a45e77
-  args:
-    - src
-    - build
-    - test
-    - --rule=indent:off
-    - --rule=no-multi-spaces:[error,{ignoreEOLComments:true}]
-    - --rule=camelcase:off # There are several `camelcase` errors in the vue codebase for some reason
+# - name: vue
```
Comment (kaicataldo) [inline projects.yml]: Waiting on  https://github.com/vuejs/vue/pull/11099 to land before we can merge this.

File: projects.yml [hunk 4]
```diff
@@ -25,60 +24,42 @@
...
 - name: redux
   repo: https://github.com/reactjs/redux
-  commit: ac7cd32a5270eb374611bf73d072e84a1e7567c2
+  commit: 9819b5ef2909cacda0f09c8d29fde229bd7da5fa
   args:
-    - src
-    - test
-    - examples
+    - --ext js,ts
+    - src/**/*
```
Comment (kaicataldo) [inline projects.yml:47]: This fails without specifying this glob.

**Comment by kaicataldo** [comment]
This is ready to go!

File: projects.yml (hunk 3 above)
Comment (kaicataldo) [reply projects.yml]: I decided to disable the failing rules and we can update later on.

**Review by btmills — APPROVED** [review]
LGTM. Nice job with all the yak shaving needed to land this.

**Review by nzakas — APPROVED** [review]

**Pull request merged**